common/eq_packet_structs.h

```diff
@@ -1793,6 +1793,17 @@ struct GMSummon_Struct {
 /*104*/	uint32	unknown2; // E0 E0 56 00
 };
 
+struct GMFind_Struct {
+/*  0*/	char	charname[64];
```

Comment:
The byte sizes are completely off in the comments (I know it was copy pasted). I would just remove them, the struct is not large enough to where someone is going to lose track counting the struct size